services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusController.kt

```diff
@@ -196,6 +196,42 @@ class HakemusController(
         return response
     }
 
+    @PostMapping("/hakemukset/{id}/laheta")
+    @Operation(
+        summary = "Send an application to Allu for processing",
+        description =
+            """Returns the application with updated status fields.
+               - Sets the pendingOnClient value of the application to false. This means the application is no longer a draft.
+               - A clerk at Allu can start processing the application after this call.
+               - The application cannot be edited after it has been sent.
+               - The caller needs to be a contact on the application for at least one customer. 
+            """
+    )
+    @ApiResponses(
+        value =
+            [
+                ApiResponse(description = "The sent application", responseCode = "200"),
+                ApiResponse(
+                    description = "Application contains invalid data",
+                    responseCode = "400",
+                    content = [Content(schema = Schema(implementation = HankeErrorDetail::class))]
+                ),
+                ApiResponse(
+                    description = "An application was not found with the given id",
+                    responseCode = "404",
+                    content = [Content(schema = Schema(implementation = HankeError::class))]
+                ),
+                ApiResponse(
+                    description = "The application has been already sent",
```

Comment:
"has already been" sounds better English.

Reply:
Yes, this was bad english. I changed it to "has been sent already" to emphasize the "already".